### ISSUE TITLE
A warning in jit_avx512_core_i8i8_pooling.cpp

### ISSUE THREAD

**Issue by luotao1** [01org/mkl-dnn#157]
There is one warning come from MKLDNN，when we compile PaddlePaddle：
```
[08:15:11] :	 [Step 1/1] [ 26%] Building CXX object src/CMakeFiles/mkldnn.dir/cpu/jit_avx512_core_i8i8_pooling.cpp.o
[08:15:11]W:	 [Step 1/1] /paddle/build/third_party/mkldnn/src/extern_mkldnn/src/cpu/jit_avx512_common_conv_kernel.cpp: In lambda function:
[08:15:11]W:	 [Step 1/1] /paddle/build/third_party/mkldnn/src/extern_mkldnn/src/cpu/jit_avx512_common_conv_kernel.cpp:2666:42: warning: assuming signed overflow does not occur when assuming that (X + c) < X is always false [-Wstrict-overflow]
[08:15:11]W:	 [Step 1/1]      auto emit_fma_block = [&](int kh_step) {
[08:15:11]W:	 [Step 1/1]                           
```
The full log is https://paddleci.ngrok.io/viewLog.html?buildId=19185&buildTypeId=Paddle_PrCi&tab=buildLog 
And would you please fix this warning later?

**Comment by rsdubtso** [comment]
Unfortunately, I cannot read the full log (need a login and a password). Can you please post the output here? Which compiler is that? Which MKL-DNN version did you use?

**Comment by luotao1** [comment]
The full log is [Paddle_PR_CI_7995.log.txt](https://github.com/01org/mkl-dnn/files/1489731/Paddle_PR_CI_7995.log.txt)
Our compiler is gcc-4.8, the MKL-DNN version is V0.11: https://github.com/PaddlePaddle/Paddle/blob/develop/cmake/external/mkldnn.cmake#L55

**Comment by rsdubtso** [comment]
This one should be fixed in 2604f435d. (In short, older GCC compilers seem to be barking at the wrong tree...).
